FACReact/App/CanMarkers.js

```diff
@@ -2,16 +2,30 @@ import React from 'react';
 import { Marker } from 'react-native-maps';
 
 function MarkCans(props) {
-  const {locations} = props;
+  const {
+    locations,
+    showGarbage,
+    showRecycling,
+    showCompost
+  } = props;
 
   return (locations) ? (
     locations.map((location) => {
       let color = 'red';
       if (location.properties.isGarbage) {
+        if (!showGarbage) {
+          return;
+        }
         color = 'tan'
       } else if (location.properties.isRecycling) {
+        if (!showRecycling) {
+          return;
+        }
         color = 'blue'
       } else if (location.properties.isCompost) {
+        if (!showCompost) {
+          return;
+        }
         color = 'green'
       }
```

Comment:
The logic of the if-statements here is unclear, could be more streamlined

Reply:
Ryan has fixed this in commit c90da8d. Thanks Ryan!